src/core/assert/dev.js

```diff
@@ -79,19 +77,18 @@ export function devAssert(
  * For more details see `assert`.
  *
  * @param {*} shouldBeElement
- * @param {!Array<*>|string=} opt_message The assertion message
- * @return {!Element} The value of shouldBeTrueish.
+ * @param {Array<*>|string=} opt_message The assertion message
```

Comment:
Didn't we discuss preferring T[] vs. Array<T>

```suggestion
 * @param {any[]|string=} opt_message The assertion message
```

Reply:
1) I'm not positive if `any` works in JSDoc comments 2) Yes, I agree, but also I haven't been updating it everywhere. Would prefer to set up TSLint and let it do it for us

Reply:
1. any does work
2. makes sense, especially if its auto-fixable.